### PR TITLE
move aiobotocore from tox to pytest

### PR DESCRIPTION
**Pull request by mabdinur**
## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: riotfile.py [hunk 1]
```diff
@@ -936,6 +936,49 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
...
+            command="pytest {cmdargs} tests/contrib/aiobotocore",
+            venvs=[
+                # aiobotocore dropped Python 3.5 support in 0.12
+                Venv(
+                    pys=["3.5", "3.6"],
+                    pkgs={
+                        "aiobotocore": [
+                            "~=0.2",
+                            "~=0.3",
+                            "~=0.4,",
```
Comment (Kyle-Verhoog) [inline riotfile.py:951]:
```suggestion
                            "~=0.4",
```

File: tox.ini [hunk 2]
```diff
@@ -260,9 +241,9 @@ deps =
     sanic: httpx==0.15.4
     sqlalchemy: sqlalchemy
     sslmodules3: aiohttp
-    sslmodules3: aiobotocore
     sslmodules: botocore
     sslmodules: requests
+    sslmodules3: aiobotocore
```
Comment (Kyle-Verhoog) [inline tox.ini:246]: nit: we can revert this change (it just got moved down)

**Review by Kyle-Verhoog — DISMISSED** [review]
just a nit otherwise lgtm

**Review by Kyle-Verhoog — DISMISSED** [review]

File: riotfile.py [hunk 3]
```diff
@@ -936,6 +936,49 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
...
             ],
         ),
+        Venv(
+            # aiobotocore: aiobotocore>=1.0 not yet supported
+            name="aiobotocore",
+            command="pytest {cmdargs} tests/contrib/aiobotocore",
+            venvs=[
+                # aiobotocore dropped Python 3.5 support in 0.12
+                Venv(
+                    pys=["3.5", "3.6"],
```
Comment (brettlangdon) [inline riotfile.py:946]: the better way to organize this is probably to subdivide by `aiobotocore` versions and not Python versions:

e.g.

- aiobotocore >= 0.2, <0.12 -> select_pys(max_version='3.5')
- aiobotocore >= 0.12 -> select_pys(min_version='3.6')

I think if I understand the way this is broken down better?

File: .circleci/config.yml [hunk 4]
```diff
@@ -785,8 +785,9 @@ jobs:
       - image: *ddtrace_dev_image
       - image: *moto_image
     steps:
-      - run_tox_scenario:
-          pattern: '^aiobotocore_contrib'
+       - run_test:
+          wait: postgres
```
[the remark this answers is not in the document]
Comment (mabdinur) [reply .circleci/config.yml]: No I thought I removed this

[the remark this answers is not in the document]
Comment (brettlangdon) [reply .circleci/config.yml]: old comment I left and never hit submit, and then  when I did this over review it submitted it 🤦🏻

**Review by brettlangdon — DISMISSED** [review]

**Review by brettlangdon — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]
